main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/joho/godotenv"
+	"go1f/pkg/api"
+	"go1f/pkg/db"
+	"go1f/pkg/server"
+	"os"
+)
+
+func main() {
+
+	_ = godotenv.Load()
+	TODO_PASSWORD := os.Getenv("TODO_PASSWORD")
+	api.PASS = TODO_PASSWORD
+
+	TODO_DBFILE := os.Getenv("TODO_DBFILE")
+	if TODO_DBFILE == "" {
+		TODO_DBFILE = "scheduler.db"
+	}
+
+	DB, err := sql.Open("sqlite", TODO_DBFILE)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Лучше так, чтобы программа завершилась с ошибкой
```suggestion
		log.Fatal(err)
```